src/test/java/com/acrolinx/sidebar/swing/AcrolinxSidebarSwingTest.java

```diff
@@ -0,0 +1,58 @@
+package com.acrolinx.sidebar.swing;
+
+import com.acrolinx.sidebar.AcrolinxIntegration;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.awt.event.KeyEvent;
+
+class AcrolinxSidebarSwingTest {
+
+    @Test
+    void processKeyEvent_PasteOnMacIsConsumed() {
+        // Given
+        KeyEvent keyEvent = Mockito.mock(KeyEvent.class);
+        AcrolinxIntegration acrolinxIntegration = Mockito.mock(AcrolinxIntegration.class);
+        AcrolinxSidebarSwing acrolinxSidebarSwing = new AcrolinxSidebarSwing(acrolinxIntegration);
+
+        // When
```

Comment:
I think this stubbing of methods here is still part of the "Given" part, the "When" part is calling `acrolinxSidebarSwing.processKeyEvent(keyEvent)` and the "Then" part the Mockito.verify() calls?

I suggest to leave out the `// Given // When // Then` comments, we can use an empty line between these logical parts to visually separate them.

Reply:
I can move it to the correct section. It makes it easier for me to understand a test case. Is it okay if I leave the comments?

Reply:
could we maybe achieve better readability with some extra help methods? For example, instead of 
`// Then
...
`
we could just call a method:
`verifyInteractionsWithMocks(keyEvent);`

Every time we need to write a comment to make the code understandable/readable, it means admitting defeat, that were unable to write the code in a clearer/better way. Since comments are not executed or verified by the compiler, they tend to become outdated/obsolete over time.

Reply:
Check out what I have just pushed, let me know if it is much better.